apps/biobin/Readme.developer.md

```diff
@@ -1,14 +1,47 @@
 
+## Examples of analysis type-specific arguments 
+(note - these are just examples; these options are not necessarily complete and should be modified according to user’s preferences):
+  - Gene binning analysis:
+  ```
+  --weight-loci Y --bin-pathways N --bin-regions Y  --bin-minimum-size 5 -F 0.01 -G 37
+  ```
+  - Pathway binning analysis:
+  ```
+  --weight-loci Y --bin-pathways Y --bin-regions N --bin-interregion N --bin-minimum-size 5 -F 0.01 -G 37
+  ```
+  - Gene binning analysis limited to only the putative functional variants using a role file:
+  ```
+  --weight-loci Y --bin-pathways N --bin-regions Y --bin-minimum-size 5 -F 0.01 -G 37 --bin-expand-roles Y --bin-expand-size 1
+  ```
+
+If the user would like to perform a permutation analysis to empirically assess the significance of the obtained results, the app (unlike the BioBin command-line application) can also be used to run a permutation analysis, in which the sample IDs in the phenotype file are being permuted. For that, the user needs to specify, in the "Permutation Count" window, how many permutations he/she would like to perform, and the app will re-run the BioBin analysis for each permuted phenotype file, keeping track of the p-values obtained from each permuted test. In addition to the regular output files described below, permutation analysis also outputs a `*permute-summary.tsv` file, which contains the empirical p-value (obtained based on individual p-values from the permuted tests) for each phenotype-bin combination tested (see below for details).
+
+## What does this app output?
+For detailed description of the output files listed below, please see the software manual.
+  - Bins report file (`*bins.csv`; one per phenotype)
+  - Summary file (`*summary.tsv`) - specific to the BioBin app, not available in the BioBin command-line application. This file contains data from the bins reports concatenated across all phenotypes.
+  - Locus file (`*locus.csv`)
+  - List of unlifted variants (`*unlifted.csv`; output only if the VCF file used as an input is based on a different genomic build than LOKI, which is currently based on build hg38).
+  - Permutation summary file (`*permute-summary.tsv`) - output only when the permutation option is enabled (i.e. when the "Permutation Count" is set to > 0 in the app settings). This output file is specific to the app, it is not available in the BioBin command-line application. This file consists of 3 columns: Outcome (i.e. phenotype name), bin (e.g. gene A), empirical p-value (which can be interpreted as the number of times out of X permutation runs that the permutation analysis returned a p-value that is lower or equal to the observed p-value, where the observed p-value is the p-value from the original BioBin run).
```

Comment:
- Permutation summary file (`*permute-summary.tsv`); output only when the permutation option is enabled (i.e. when the "Permutation Count" is set to > 0 in the app settings). This output file is specific to the app, it is not available in the BioBin command-line application. This file consists of 3 columns: `outcome` (i.e. phenotype name), `bin` (e.g. gene A), `empirical p-value` (which can be interpreted as the number of times out of X permutation runs that the permutation analysis returned a p-value that is lower or equal to the observed p-value, where the observed p-value is the p-value from the original BioBin run).